### PR TITLE
De-lint recipes.

### PR DESCRIPTION
**Pull request by Max-Might**
I had some free time today and decided to help with this.

### CODE REVIEW

File: games-arcade/pachi/pachi-1.0.recipe [hunk 1]
```diff
@@ -1,6 +1,6 @@
-SUMMARY="Pachi el marciano is a cool 2D platforms game"
-DESCRIPTION="Pachi el marciano is a cool 2D platforms game inspired in games like \
-Manic Miner, Jet set Willy and more from the 80s." 
+SUMMARY="A cool 2D platform game"
```
Comment (waddlesplash) [inline games-arcade/pachi/pachi-1.0.recipe]: Should be `platforming`.

File: media-libs/opus/opus-1.1.recipe [hunk 2]
```diff
@@ -1,19 +1,19 @@
-SUMMARY="Opus Interactive Audio Codec"
-DESCRIPTION="
-Opus is a totally open, royalty-free, highly versatile audio codec. Opus is \
+SUMMARY="An Interactive Audio Codec"
```
Comment (waddlesplash) [inline media-libs/opus/opus-1.1.recipe]: Sentence casing should be used here.

**Comment by Max-Might** [comment]
@waddlesplash Fixed, thanks.  I also added one more commit.

**Comment by Max-Might** [comment]
I was not sure if I wanted to keep it.
There is a better recipe for version 1.13. Should I delete the 1.12 one?

**Comment by waddlesplash** [comment]
Oh -- yes, if there is a newer recipe, delete the old one. And squash your commits.

**Comment by Max-Might** [comment]
I deleted the recipe and patch for 1.12 and squashed the commits.

**Comment by waddlesplash** [comment]
Thanks!

File: games-arcade/pachi/pachi-1.0.recipe (hunk 1 above)
Comment (pulkomandy) [inline games-arcade/pachi/pachi-1.0.recipe]: https://en.wikipedia.org/wiki/Platform_game

I never heard "platforming game" used anywhere?

**Pull request merged**